dist/js/index.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+var _starter_app = require('./starter_app');
+
+var _starter_app2 = _interopRequireDefault(_starter_app);
```

Comment:
I see that you are using '_' as first character in most var and function names, does it mean its private or is it just naming convention?